bigtop-tests/smoke-tests/hive/HiveJdbcGeneralTest.java

```diff
@@ -0,0 +1,87 @@
+
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.apache.hadoop.fs.Path;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+//Author: Bharat Modi
+
+
+
+public class HiveJdbcGeneralTest extends TestMethods {
+
+	@Test
+	public void testTableCreation() throws SQLException,
+	ClassNotFoundException, InstantiationException,
+	IllegalAccessException, IOException, URISyntaxException,
+	FileNotFoundException, ParserConfigurationException, SAXException {
+		final File f = new File(HiveJdbcGeneralTest.class.getProtectionDomain().getCodeSource().getLocation().getPath());

```

Comment:
Length of the lines should be kept under 80, if possible. Please check
"REVIEWING PATCHES" section from https://cwiki.apache.org/confluence/display/BIGTOP/How+to+Contribute for more details.